adapters/faraday/test/opentelemetry/adapters/faraday/middlewares/tracer_middleware_test.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'test_helper'
+
+# require Adapter so .install method is found:
+require_relative '../../../../../lib/opentelemetry/adapters/faraday/adapter'
+require_relative '../../../../../lib/opentelemetry/adapters/faraday/middlewares/tracer_middleware'
+
+describe OpenTelemetry::Adapters::Faraday::Middlewares::TracerMiddleware do
```

Comment:
Just comparing this test to the data dog version: https://github.com/DataDog/dd-trace-rb/blob/master/spec/ddtrace/contrib/faraday/middleware_spec.rb. and there's much less coverage here: 

I assume that we do not have all of the same functionality, so less tests are expected. Do we have a corresponding test for all common functionality?

Reply:
Most of those tests are for configuration overrides, distributed context, overriding service name, opt-in/opt-out request header tracing... a lot of functionality that doesn't exist in this version.

I think we do have decent coverage for the functionality that does exist.

Reply:
Thanks for sifting through those and double checking!